tests/test_server.py

```diff
@@ -104,9 +106,11 @@ def setUpClass(cls):
             configFile=os.path.join(TESTDIR, "config/basic/butler-s3store.yaml"),
             forceConfigRoot=False,
         )
-
         # Add a file with corrupted data for testing error conditions
         cls.dataset_with_corrupted_data = _create_corrupted_dataset(cls.repo)
+        # All of the datasets that come with MetricTestRepo are disassembled
```

Comment:
I had forgotten that the MetricTestRepo defaults to storage class `StructuredCompositeReadComp`.  Easy to change the default if we wanted to but also nice to know that getURIs should work.